astroid/brain/brain_stdlib.py

```diff
@@ -437,3 +446,4 @@ def shutdown(self):
 register_module_extender(MANAGER, 'multiprocessing.managers',
                          multiprocessing_managers_transform)
 register_module_extender(MANAGER, 'multiprocessing', multiprocessing_transform)
+register_module_extender(MANAGER, 'threading', thread_transform)
```

Comment:
Sorry. You're right but I'm running out of time... Would it be OK if I modify the module transform to have Lock being a function?